### PR TITLE
docs(README): add TLDR section

### PR DESCRIPTION
**Pull request by champignoom**
For [neorg users](https://discord.com/channels/834325286664929280/834326555160608769/1148067195755175936) that are not neovim devs or debuggers.

Hopefully this caps the counterproductivity of configuring for productivity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,82 @@ To learn more about the philosophy of the project check the [philosophy](#-philo
 
 **Neorg requires at least Neovim 0.8+ to operate.**
 
+### TL;DR
+
+For neovim beginners who don't want to tinker with the configurations:
+
+1. Install [Meslo Nerd Font](https://github.com/ryanoasis/nerd-fonts/).
```
Comment (max397574) [inline README.md:78]:
```suggestion
1. Install [Meslo Nerd Font](https://www.nerdfonts.com/font-downloads) or any other nerd font.
```

File: README.md [hunk 2]
```diff
@@ -71,6 +71,82 @@ To learn more about the philosophy of the project check the [philosophy](#-philo
...
+  
+  vim.g.mapleader = " "
+  
+  require("lazy").setup({
+    "rebelot/kanagawa.nvim",  -- neorg needs a colorscheme with treesitter support
+    {
+      "nvim-treesitter/nvim-treesitter",
+      build = ":TSUpdate",
+      opts = {
+        ensure_installed = { "lua" },
```
Comment (max397574) [inline README.md:111]:
```suggestion
        ensure_installed = { "lua", "norg" },
```
I'm not even sure if lua is still needed. Isn't it bundled in core now?

File: README.md [hunk 3]
```diff
@@ -71,6 +71,82 @@ To learn more about the philosophy of the project check the [philosophy](#-philo
...
+      },
```
Comment (max397574) [inline README.md:112]:
```suggestion
        highlight = { enable = true },
      },
```

File: README.md (hunk 2 above)
Comment (max397574) [reply README.md:111]: nvim the norg, since we have the "sync-parsers" command

**Review by max397574 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (champignoom) [reply README.md:78]: The github page loads faster and looks more instructive.

The reason that I put one particular font here is that people might struggle with figuring out the exact character sequence to put in ther config file (here it's "MesloLGM Nerd Font Mono"). For those who actually have a preference outside Meslo, it should be self-evident anyway that Meslo isn't the only choice.

File: README.md (hunk 2 above)
Comment (champignoom) [reply README.md:111]: Isn't sync-parsers taking care of norg parser already?

If not, I guess we also need norg_meta.

I'm not sure if lua is need. `opts = {}` is probably all we need ?

**Review by max397574 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -71,6 +71,83 @@ To learn more about the philosophy of the project check the [philosophy](#-philo
 
 **Neorg requires at least Neovim 0.8+ to operate.**
 
+### TL;DR
+
+For neovim beginners who don't want to tinker with the configurations:
+
+1. Install [Meslo Nerd Font](https://github.com/ryanoasis/nerd-fonts/).
+2. Set your terminal font to "MesloLGM Nerd Font Mono".
+3. Make sure you have git.
```
Comment (max397574) [inline README.md:80]: users who don't realize that they should have git also perhaps don't know if they have it or not
```suggestion
3. Make sure you have git by running `git --version`
```

**Comment by max397574** [comment]
forgot the `.` sorry😅

**Comment by champignoom** [comment]
A minimal working configuration like this has been [proved](https://github.com/nvim-neorg/neorg/issues/1050#issuecomment-1705902049) to be effective for debugging as well.

**Pull request merged**